### PR TITLE
Add BalenaOS support for ODYSSEY - X86 Boards

### PR DESCRIPTION
**Pull request by lakshanthad**

### CODE REVIEW

File: layers/meta-balena-odyssey-x86/classes/odyssey-image.bbclass [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+FEATURE_PACKAGES_nfs-server = "packagegroup-core-nfs-server"
+FEATURE_PACKAGES_nfs-client = "packagegroup-core-nfs-client"
+FEATURE_PACKAGES_ssh-server-dropbear = "packagegroup-core-ssh-dropbear"
+FEATURE_PACKAGES_ssh-server-openssh = "packagegroup-core-ssh-openssh"
+FEATURE_PACKAGES_hwcodecs = "${MACHINE_HWCODECS}"
+
+
+# IMAGE_FEATURES_REPLACES_foo = 'bar1 bar2'
+# Including image feature foo would replace the image features bar1 and bar2
+IMAGE_FEATURES_REPLACES_ssh-server-openssh = "ssh-server-dropbear"
```
Comment (floion) [inline layers/meta-balena-odyssey-x86/classes/odyssey-image.bbclass:57]: Why is this needed here?

**Comment by floion** [comment]
Hi, I think you have forgot to add the following to your PR: https://github.com/balena-os/balena-seeed-x86#pull-requests (so a Changelog-entry and so on)

File: layers/meta-balena-odyssey-x86/classes/odyssey-image.bbclass (hunk 1 above)
Comment (lakshanthad) [reply layers/meta-balena-odyssey-x86/classes/odyssey-image.bbclass:57]: This is not needed. It is my mistake. I will delete it

**Comment by lakshanthad** [comment]
> Hi, I think you have forgot to add the following to your PR: https://github.com/balena-os/balena-seeed-x86#pull-requests (so a Changelog-entry and so on)

I will do this, delete "odyssey-image.bbclass" as mentioned above, and then redo this PR. 
Thank you.

**Comment by lakshanthad** [comment]
https://github.com/balena-os/balena-seeed-x86/pull/4

**Comment by floion** [comment]
>
